### PR TITLE
Feature/improve readme

### PR DESCRIPTION
**Pull request by KieranFrance**

### CODE REVIEW

**Review by markee174 — APPROVED** [review]

**Comment by KieranFrance** [comment]
I have made a couple more changes based on universal changes suggested in the related reviews

File: README.md [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
-```
-cd docker
-docker build -t idrsolutions/jpedal . 
+JPedal is a Java PDF library for converting, extraction, viewing and printing PDF documents. This docker image can be used to containerise JPedal making its image/text extraction and PDF to image conversion functionality accessible via a REST API which is perfect for cloud deployments.
+
+## Getting Started ##
+
+In order to use the JPedal Docker image you will need a license to access the JPedal war file. If you have not got a license yet, you can find our [free trial or contact us here](https://www.idrsolutions.com/jpedal/pricing).
+
+Once you have access, you can get and run the docker image with the following commands.
```
Comment (leonatherton) [inline README.md]:
```suggestion
Once you have the JPedal war file, you can pull and run the docker image with the following commands:
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
 
-## Build image
-```
-cd docker
-docker build -t idrsolutions/jpedal . 
+JPedal is a Java PDF library for converting, extraction, viewing and printing PDF documents. This docker image can be used to containerise JPedal making its image/text extraction and PDF to image conversion functionality accessible via a REST API which is perfect for cloud deployments.
+
+## Getting Started ##
+
+In order to use the JPedal Docker image you will need a license to access the JPedal war file. If you have not got a license yet, you can find our [free trial or contact us here](https://www.idrsolutions.com/jpedal/pricing).
```
Comment (leonatherton) [inline README.md]:
```suggestion
In order to use the JPedal Docker image you will need a license to access the JPedal war file. If you have not got a license yet, you can [sign up for a free trial](https://www.idrsolutions.com/jpedal/trial-download).
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,28 @@
...
+- Clone the project from [here](https://github.com/idrsolutions/jpedal-docker)
+- Navigate to the project directory in a terminal
+- Run the following command  
+  ```docker build -t idrsolutions/jpedal .```
+
+## Documentation ## 
 
-## Run container
-Full details on how this Docker Image can be set up and used can be [found here](https://support.idrsolutions.com/jpedal/tutorials/cloud/docker/deploy-jpedal-on-docker).
+[IDRSolutions JPedal Cloud Support Page](https://support.idrsolutions.com/jpedal/tutorials/cloud/).
```
Comment (leonatherton) [inline README.md]:
```suggestion
[JPedal Cloud Documentation](https://support.idrsolutions.com/jpedal/tutorials/cloud/)
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,28 @@
...
+- Navigate to the project directory in a terminal
+- Run the following command  
+  ```docker build -t idrsolutions/jpedal .```
+
+## Documentation ## 
 
-## Run container
-Full details on how this Docker Image can be set up and used can be [found here](https://support.idrsolutions.com/jpedal/tutorials/cloud/docker/deploy-jpedal-on-docker).
+[IDRSolutions JPedal Cloud Support Page](https://support.idrsolutions.com/jpedal/tutorials/cloud/).  
+[IDRSolutions JPedal Docker Support Page](https://support.idrsolutions.com/jpedal/tutorials/cloud/docker).
```
Comment (leonatherton) [inline README.md]:
```suggestion
[JPedal Docker Documentation](https://support.idrsolutions.com/jpedal/tutorials/cloud/docker)
[Contact IDRsolutions](https://www.idrsolutions.com/contact-us)
```

**Review by leonatherton — CHANGES_REQUESTED** [review]

**Comment by KieranFrance** [comment]
The suggested changes have now been implemented

**Review by leonatherton — APPROVED** [review]

**Pull request merged**